packages/acs-ui-common/src/identifier.ts

```diff
@@ -60,3 +60,10 @@ export const _isValidIdentifier = (identifier: CommunicationIdentifier): boolean
     isUnknownIdentifier(identifier)
   );
 };
+
+/**
+ * Check if given identifier is a Microsoft Teams user.
+ *
+ * @internal
+ */
+export const isIdentityMicrosoftTeamsUser = isMicrosoftTeamsUserIdentifier;
```

Comment:
Why are we wrapping this rather than just using the library directly?

Reply:
There are some wrapping functions above showing how underlying functionality can be combined to provide more useful functions. They also have the preceding underscore in the name to mark them as internal.